bin/generate_fake_data.rb

```diff
@@ -0,0 +1,111 @@
+#!env ruby
+
+require "factory_bot"
+require "faker"
+require_relative "../spec/fixtures/organizations"
+require_relative "../spec/fixtures/collections"
+
+FactoryBot.find_definitions
+
+Services.register(:ht_organizations) { mock_organizations }
+Services.register(:ht_collections) { mock_collections }
+
+OUTDIR = "/usr/src/app/testdata"
+MAX_OCN = 100000
+
+def random_year
+  Faker::Date.between(from: Date.parse("1850-01-01"), to: Date.today).year
+end
+
+def warn_exists(outfile)
+  if(File.exist?(outfile))
+    puts "#{outfile} already exists, not creating"
+    true
+  else
+    false
+  end
+end
+
+
+# For now, only single-part monographs
+
+def generate_holdings(count = 50000, inst = "umich", date = Date.parse("2020-01-01"))
+  # holdings
+  outfile = File.join(OUTDIR, "umich_fake_holdings.ndj")
```

Comment:
Might want to parameterize the file output so that we can load holdings from multiple members.

It was much easier to build holdings in JSON format than to deal with the TSV format. Eventually we may want to be able to generate fake data in the format members are expecting to send us so that we can do end-to-end testing with autoscrub.